src/main/groovy/org/labkey/gradle/plugin/Distribution.groovy

```diff
@@ -233,10 +232,12 @@ class Distribution implements Plugin<Project>
     {
         if (project.dist.artifactId != null)
             return project.dist.artifactId
-        else if (project.tasks.findByName("distribution") != null)
+        else
         {
-            if (project.tasks.distribution instanceof ModuleDistribution)
-                return ((ModuleDistribution) project.tasks.distribution).getArtifactId()
+            TaskUtils.doIfTaskPresent(project, "distribution", (distTask) -> {
+                if (distTask.get() instanceof ModuleDistribution)
+                    return ((ModuleDistribution) distTask).getArtifactId()
+            })
```

Comment:
I don't think this will get returned from `getArtifactId`
```suggestion
            return TaskUtils.getOptionalTask(project, "distribution")
                    .filter(task -> task.get() instanceof ModuleDistribution)
                    .map(task -> ((ModuleDistribution)task.get()).getArtifactId())
                    .orElse(project.name)
```